host/src/lib.rs

```diff
@@ -106,6 +106,10 @@ pub struct Cli {
     #[serde(flatten)]
     /// Proof request options
     pub proof_request_opt: ProofRequestOpt,
+
+    #[arg(long, require_equals = true)]
```

Comment:
give a default 123456/taiko/whatever maybe easier for script test.

Reply:
yes I think either we need some dummy data so people don't have to set things up to be able to quickly prove blocks, or maybe we could make the jwt check optional by doing `jwt_secret: Option<String>` and only checking it if it's set?

Reply:
Set as `option`, without default value